node/blockchain/facade.py

```diff
@@ -31,11 +33,34 @@ def clear_instance_cache(cls):
         cls._instance = None
 
     def get_next_block_number(self) -> int:
-        # TODO(dmu) CRITICAL: Implement with write-through cache
-        #                     https://thenewboston.atlassian.net/browse/BC-154
-        return 1
+        # TODO(dmu) HIGH: Implement method via write-through cache
+        #                 https://thenewboston.atlassian.net/browse/BC-175
+        from node.blockchain.models import Block
+        last_block = Block.objects.get_last_block()
+        return last_block._id + 1 if last_block else 0
 
     def get_next_block_identifier(self) -> BlockIdentifier:
-        # TODO(dmu) CRITICAL: Implement with write-through cache
-        #                     https://thenewboston.atlassian.net/browse/BC-154
-        return BlockIdentifier('fa1e' * 16)
+        # TODO(dmu) HIGH: Implement method via write-through cache
+        #                 https://thenewboston.atlassian.net/browse/BC-175
+        from node.blockchain.models import Block
+        last_block = Block.objects.get_last_block()
+        return last_block.get_message().make_hash() if last_block else None  # Genesis block has identifier of `null`
```

Comment:
@buckyroberts we are using block message (not including signer and signature) to calculate the next block identifier. Should include signer and signature in hashing as well?